lib/inspec/resources/selinux.rb

```diff
@@ -0,0 +1,57 @@
+require "inspec/resources/command"
+
+module Inspec::Resources
+  class Selinux < Inspec.resource(1)
+    name "selinux"
+
+    supports platform: "unix"
```

Comment:
This can be changed to "linux" and it will automatically handle detecting specifically linux (not general unix) platforms.